tools/crm_resource.c

```diff
@@ -1644,6 +1593,58 @@ handle_locate(pcmk_resource_t *rsc)
     return rc;
 }
 
+static int
+handle_metadata(void)
+{
+    int rc = pcmk_rc_ok;
+    char *standard = NULL;
+    char *provider = NULL;
+    char *type = NULL;
+    char *metadata = NULL;
+    lrmd_t *lrmd_conn = NULL;
+
+    rc = lrmd__new(&lrmd_conn, NULL, NULL, 0);
+    if (rc != pcmk_rc_ok) {
+        g_set_error(&error, PCMK__RC_ERROR, rc,
+                    _("Could not create executor connection"));
+        lrmd_api_delete(lrmd_conn);
+        return rc;
+    }
+
+    rc = crm_parse_agent_spec(options.agent_spec, &standard, &provider, &type);
+    rc = pcmk_legacy2rc(rc);
+
+    if (rc == pcmk_rc_ok) {
+        rc = lrmd_conn->cmds->get_metadata(lrmd_conn, standard,
+                                           provider, type,
+                                           &metadata, 0);
+        rc = pcmk_legacy2rc(rc);
+
+        if (metadata) {
```

Comment:
Likewise with best practices here... you could compare directly to NULL.

Reply:
For sure. Again, just I usually copy-paste when moving code around, but I'll update this commit as it's no big deal.